uportal-war/src/main/resources/properties/i18n/Messages.properties

```diff
@@ -391,6 +391,7 @@ most.frequently.added=Most Frequently Added
 move=Move
 move.this.column=Move this column
 move.this.portlet=Move this portlet
+move.this.portlet.error=This change was disallowed by the server.  The portlet will return to its original position.  Reload the page now?
```

Comment:
This is neat, curious if the typical users would get confused by this. Maybe "You cannot move this portlet"? What would be really neat if then it moved it back, or even better if it didn't allow it to move in the first place. There could be something passed in from XSL saying whether or not it is movable. I remember being able to set something in the fragment layout. The move back option would be just capturing what was before it initially, then if you get the forbidden, then just run an "undo" dom change.